jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/error/ExecutionErrorHandlerImpl.java

```diff
@@ -18,13 +18,10 @@
 
 import java.util.List;
 
+import org.apache.maven.execution.ExecutionListener;
```

Comment:
is this the right ExecutionListener? or do we actually need it....

Reply:
shouldn't be there :)